src/client/Client.cc

```diff
@@ -4265,7 +4265,7 @@ void Client::signal_cond_list(list<ceph::condition_variable*>& ls)
   }
 }
 
-void Client::wait_on_context_list(list<Context*>& ls)
+void Client::wait_on_context_list(std::vector<Context*>& ls)
```

Comment:
We have already brought `std::vector` into the current scope at line 159 `using std::vector;`

Reply:
I prefer explicit namespacing (and we generally do so everywhere).

Reply:
so should we remove the import at line 159?

Reply:
in a separate PR, sure.